README.md

```diff
@@ -10,9 +10,9 @@ Shipyard is an Entity Component System focused on usability and speed.
 If you have any question or want to follow the development more closely join the [Zulip](https://shipyard.zulipchat.com).
 
 There's two big learning resources:
-- (Soon™) The Tutorial for people new to ECS or preferring to learn by making a project.
+- (Soon™) The Tutorial for people new to ECS or who prefer to learn by making things.
```

Comment:
I agree with the `who`, however I'd suggest:

> (Soon™) The Tutorial for people who are new to ECS or prefer to learn by making a project.

In terms of "things" vs. "a project", it's a bit nitpicky, but I prefer "a project"... "things" feels a little too vague. I don't feel strongly about that at all though, either way

Reply:
I agree with @dakom, maybe we'll find a better way to end the sentence but I prefer "a project" to "things".
```suggestion
- (Soon™) The Tutorial for people new to ECS or who prefer to learn by making a project.
```